### ISSUE TITLE
Window resize issue when heading is too long

### ISSUE THREAD

**Issue by mcloota** [gristlabs/mkdocs-windmill#43]
Another minor issue I noticed is that when I type a heading that is too long (they are nested within other headings btw) the window resize makes the navigation window on the left larger and spill out onto the notes section of the pages.

Example:
in a .md file, I use a very long heading:
```
## Very extremely super duper longish but pretty long heading
```
- Try recreating it and see if it's just me.

I'm using Chrome on a fairly new Macbook Pro 13"

**Comment by dsagal** [comment]
Yeah, this seems a problem. I recommend using `extra_css` (described at https://www.mkdocs.org/user-guide/configuration/#extra_css) and add there:

```
.wm-toc-pane {
  max-width: 250px;
}
```
